### ISSUE TITLE
JAWS does not announce the role or label for group/radiogroup/toolbar/grid in dialog with virtual cursor off

### ISSUE THREAD

**Issue by kabsas** [FreedomScientific/VFO-standards-support#706]
### Summary
JAWS does not announce the role or label for group/radiogroup/toolbar/grid in dialog with virtual cursor off

Steps to reproduce:
1. Launch JAWS
2. Go to https://codepen.io/yououhan123/pen/rNZdgvP
3. Turn off virtual cursor (Ctrl+Z)
4. Navigate to Open dialog button and activate the button
5. Navigate through the elements in the dialog

### Expected result

* When focus moves to Button 1 (or Button 2 when navigating backwards), the toolbar role and label (My toolbar) is announced
* When focus moves to Button 3 (or Button 2 when navigating backwards), the group role and label (My group) is announced
* When focus moves to the Row 1 Column 1 button (or the last row/column button when navigating backwards), the grid role and label (My grid) is announced
* When focus moves to the check radio button, the radiogroup role and label (Contact:) is announced

### Actual result

* When focus moves to Button 1 (or Button 2 when navigating backwards), the toolbar role and label (My toolbar) are not announced
* When focus moves to Button 3 (or Button 2 when navigating backwards), the group role and label (My group) are not announced
* When focus moves to the Row 1 Column 1 button (or the last row/column button when navigating backwards), the grid role and label (My grid) are not announced
* When focus moves to the check radio button, the radiogroup role and label (Contact:) are not announced

#### JAWS version and build number
2023.2302.15
 
#### Operating System and version
Windows 10 Enterprise 19044.2604

#### Browser and version:
Chrome 111
Edge 111
Firefox 110

**Comment by JAWS-test** [comment]
I can confirm the error. Occurs only in conjunction with role=dialog and manual deactivation of the virtual cursor (and not when role=application is used).

**Comment by stevefaulkner** [comment]
@kabsas what is the use case for turning off virtual cursor?
